content/numpy/concepts/math-methods/terms/cos/cos.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: 'NumPy cos'
+Description: 'Returns the cos of the argument.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'NumPy'
+  - 'Trig'
+  - 'Math'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+In NumPy, the **cos** function computes the cosine of each element in an array. This trigonometric function is essential for various mathematical computations, especially in physics, engineering, computer graphics, and signal processing, and more.
+
+## Syntax
+
+The generic syntax for the NumPy math methods is as follows:
+```pseudo
+numpy.cos(argument)
```

Comment:
This is not the correct syntax, please re-check this once, the correct syntax is:
`numpy.cos(array, out = None, dtype = None)`